.github/workflows/scripts/install-go.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
```

Comment:
Instead do
```
set -eu
set -x
```

where the `set -x` could get dropped at a later date. And our scripts should always fail on unset vars